src/components/user/UserReservations.vue

```diff
@@ -6,35 +6,44 @@ import BaseCard from "@/components/base/BaseCard.vue";
 import ReservationItem from "@/components/reservations/ReservationItem.vue";
 import LoadingSpinner from "@/components/LoadingSpinner.vue";
 import NoResults from "@/components/NoResults.vue";
-import type { ShowTicket } from "@/types/reservations";
+import type { ShowTicket, Reservation } from "@/types/reservations";
 const userStore = useUserStore();
 
-const userReservations = ref();
-const ticketList = ref<ShowTicket[]>([]);
+const userReservations = ref<[] | Reservation[]>();
+const ticketList = ref<[] | ShowTicket[]>([]);
```

Comment:
I don't think you need `[] |` here

Reply:
Include n on next branch, will apply this here